google-cloud-bigquery/benchmark/inserts.rb

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/bigquery"
+require "benchmark"
+require "securerandom"
+require "time"
+require "date"
+
+if ARGV.length < 1
+  puts "usage: BIGQUERY_PROJECT=<project-id> ruby benchmark/inserts.rb 2000"
```

Comment:
Want to copy this usage doc to [benchmark/README.md](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-bigquery/benchmark/README.md)?

Reply:
Updated.